ui/src/actions/remoteAccess.js

```diff
@@ -41,57 +36,51 @@ export function requestControl(
   name = '',
   userInfo = {},
 ) {
-  return () => {
-    sendRequestControl(control, message, name, userInfo);
-  };
+  return () => sendRequestControl(control, message, name, userInfo);
```

Comment:
By returning the promise here, I turn the arrow function into an asynchronous arrow function (there's a linting rule to prevent adding `async` without `await`, so it's not super obvious). Returning promises from thunks allows awaiting `dispatch` calls - e.g. 

```js
await dispatch(requestControl(...))
// do something after the `sendRequestControl` promise has fullfilled
```